README.md

```diff
@@ -80,9 +80,14 @@ Generate MaaS compatible image
 
 Generating an image for MaaS follows the same rules outlined above for OpenStack images. For this purpose there is a separate commandlet:
 
+Before using the scripts make sure you have the proper ExecutionPolicy:
```

Comment:
Change this to something along the lines of: if you download this repo as a zipfile on windows, make sure you ```Unblock-File repo.zip``` before unarchiving. This will prevent errors about scripts not being signed.